packages/ui-components/src/components/kv-breadcrumb/kv-breadcrumb.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, h, Prop, Host, Event, EventEmitter, Fragment, Listen } from '@stencil/core';
+import { IBreadcrumbItem } from '../kv-breadcrumb-item/kv-breadcrumb-item.types';
+
+@Component({
+	tag: 'kv-breadcrumb',
+	shadow: true,
+})
+export class KvBreadcrumb {
+	/** (required) List of breadcrumbs with the last being the current path */
```

Comment:
the current path is controlled with  the `active` flag and not the position in the array. fix the comment only